_opensearch/rest-api/snapshots/create-repository.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Create Snapshot Repository
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+## Create snapshot repository
+
+Registers a new repository to store snapshots. Repositories comprise two types:
+
+* File system (`fs`)
+
+* Amazon S3 bucket (`s3`)
+
+To learn more about repositories, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
+
+
+### Path parameters
+
+Parameter | Data Type | Description
+:--- | :--- | :---
+repository | String | Repostory name. |
+
+
+### Query parameters
+
+None
+
+#### Sample request
+
+You can create repositories with `POST` or `PUT`. 
+{: .note}
+
+The following requests create a repository called `my-first-repo`:
+
+`POST _snapshot/my-first-repo/`
+
+`PUT _snapshot/my-first-repo/`
+
+
+#### Sample response
+
+Upon success, the response returns the following JSON object:
+
+```json
+{
+  "acknowledged": true
+}
+```
+
+To verify that the repository was created, use the [Get snapshot repository API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/snapshots/get-snapshot-repository), passing the repository name as the `repository` path parameter.
+{: .note}
+
+### Request body
+
+Allowable parameters depend on the repository type.
+
+#### fs repository type
```

Comment:
I would make fs repository type and s3 repository type headings, and provide sample requests and responses under each heading.